### ISSUE TITLE
Default focus ring can be hard to see

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#1201]
As shown in this screenshow, sometimes the default focus ring color is hard to distinguish:

![2ab5ab15d776e7a8](https://user-images.githubusercontent.com/283842/57463954-a7c1bf80-7230-11e9-994a-9c39e87c451d.png)

I think we probably just need to make the default focus ring color a bit darker. Or maybe it should be thicker in certain contexts, such as the post privacy dialog? I'm not 100% sure.

**Comment by notabene** [comment]
@nolanlawson  I like the (perhaps too obvious but very efficient) outline for focus on [WAI's website](https://www.w3.org/WAI/). It's an inspiraiton I'll draw from for a future redesign on my website.

As promised, I'll test the other themes and let you know if one is better than the other.

Thanks!

**Comment by nolanlawson** [comment]
Thanks! It looks like that site uses a 2px solid outline with a 2s transition on outline-offset so that it animates when you focus to it. I think that's a bit excessive for my taste, but I think probably the contrast could be improved, and maybe the outline-offset could be tweaked.

**Comment by nolanlawson** [comment]
I tried the WAI version of an outline that animates on focus, and I think it's quite a bit of overhead (have to modify styles on `*` to add a transparent outline everywhere to make the animation work, animating `outline-offset` itself can't be GPU-accelerated), and plus I found it a bit nauseating, and I don't even have any vestibular disorders, so I would definitely have to disable it for `prefers-reduced-motion: reduce`.

I think a more reasonable fix would be a targeted one. The issue here is that the focus outline is getting cropped inside of that particular button. We can just fix those buttons.

**Comment by notabene** [comment]
I love your thoroughness! Thanks a lot!
